guide/case-studies/saving-satoshi.md

```diff
@@ -0,0 +1,433 @@
+
+1. Beginner developer with 1 year of experience with little to no experience working on bitcoin  
+2. Senior developer with little to no experience working on bitcoin  
+3. Experienced bitcoin developer
+
+The scope was to test only one chapter per interview. To avoid influencing them, we asked open-ended questions and quietly observed how they completed the challenges.
+
+We also tested a variety of hypotheses across the verticals of demographics, UI, UX, and lesson content such as:
+
+- Demographics: This game is engaging enough for Senior Developers to continue playing.  
+- UI: Tooltip pop-ups triggered by a hovering mouse are intuitive.  
+- UX: This clue that appears after three failed challenge attempts will help the user move on to the next step.  
+- Lesson content: This topic and challenge won’t be too difficult for a beginner developers.
+
+These interviews were then summarized and shared with the team.
+
+### In-Person Feedback
+
+Fourth, we obtain mass feedback from events such as TABConf. During that event we led a workshop where roughly 50 people played the game at the same time. All the while, the product team was there to answer questions, address concerns, and gather feedback. This event also helped stress test the system and it was revealed that our infrastructure at the time was not able to handle a certain amount of load.
+
+### Dog-fooding
+
+Lastly, the product team that built this game also happens to be the kind of user that we’re building it for. By replaying the game on our own, we provided ongoing, iterative feedback to each other about what delighted and what frustrated us. 
+
+All in all, the product team had several different sources of data and feedback about Saving Satoshi. This was used ongoing to inform features and ideas.
+
+## What’s next
+
+Chapter 10 may be a major milestone for Saving Satoshi, but the project is far from over! At the time of writing, December 2024, there are still many things the team has planned for future releases. From design, to learning content, to community, these ideas are not confined to just one aspect of the project and instead span across many facets of the game.
+
+**Difficulty levels**
+One of the larger items on the to-do list is support for difficulty levels. In an ideal world, Saving Satoshi would be able to accommodate a variety of learners but currently, a solid understanding of how to code is a major requirement. It would be nice to one day support people that are just getting started and are new to coding. Conversely, it would also be great to make advanced exercises for those looking for more of a challenge. This has been implemented to some degree by sometimes moving hints to a separate help page, but there is so much more that can be done. At the moment, chapter six supports two difficulty modes (normal and hard). This was initially implemented as a proof-of-concept and the team looks forward to extending this functionality to other chapters.
+
+**Movie trailer**
```

Comment:
One more - Same issue here - I'm not sure why but there is no line break after this in the preview